lib/ansible/modules/system/syspatch.py

```diff
@@ -113,7 +113,8 @@ def syspatch_run(module):
     reboot_needed = False
     warnings = []
 
-    # Setup command flags
+    run_flag = []
+    check_flag = []
```

Comment:
Similarly, if `check_flag` is `[]` when the module uses it:

https://github.com/ansible/ansible/blob/34ecd6cb25258116d394ffcbaf9ec29504a3919a/lib/ansible/modules/system/syspatch.py#L128-L129

Then `syspatch(8)` will apply all available patches. Set this to `-c` or `-l` to prevent that. If the user specifies `revert` then the module sets `check_flag` to `['-l']`, so I'll suggest `['-c']` for the default:

```suggestion
    check_flag = ['-c']
```